system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -226,6 +226,23 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 0;
 }
 
+pub struct BaseFilter;
+impl Contains<RuntimeCall> for BaseFilter {
+	fn contains(call: &RuntimeCall) -> bool {
+		// Disallow these Snowbridge system calls.
+		if matches!(
+			call,
+			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_agent { .. })
+		) || matches!(
+			call,
+			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_channel { .. })
+		) {
```

Comment:
Just personal preference, this looks nicer.
```suggestion
			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_agent { .. }) |
				RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_channel { .. })
		) {
```